### ISSUE TITLE
[REQUEST] In game tablet

### ISSUE THREAD

**Issue by Noah-Huppert** [flybywiresim/a32nx#947]
**Is your feature request related to a problem? Please describe.**

As a user I would like to view charts in game without alt + tab -ing onto my desktop.

**Describe the feature you'd like to see implemented**

Initially just some way of viewing charts and such. Doesn't even need a chart searching interface, I would be content with needing to drop PDF files into a folder before starting my flight, and the tablet only showing those. Stretch goal would be something like Avitab's Navigraph integration.

**References**

None, I think making a EFB is in line with the mission of this mod. Exactly replicating an existing EFB solution would be cool, however I think just getting some sort of EFB would be nice for now.

**Additional context**

I am trying to make this right now, I am making this issue to track my efforts and ensure duplicate work does not occur within the project.

Discord username (if different from GitHub): `TurtleKayak`.

**Comment by GlitchedYT** [comment]
We already have an EFB modelled and ready, only issue is there's no way to import it as of now,

**Comment by Noah-Huppert** [comment]
Oh okay, glad to hear. I'll close this then.
